install/streams.csv

```diff
@@ -2559,7 +2559,7 @@ TROB,28,H,N,200,XYZ,false,true,2013-08-02T00:00:00Z,2019-12-16T00:00:00Z
 TROB,28,H,N,200,XYZ,false,false,2019-12-16T00:00:01Z,9999-01-01T00:00:00Z
 TRTS,20,B,N,50,true,false,false,2012-02-10T00:00:10Z,2020-06-16T00:00:00Z
 TRTS,20,H,N,200,true,false,true,2012-02-10T00:00:10Z,9999-01-01T00:00:00Z
-TRVZ,10,L,H,1,false,false,false,2009-04-14T22:00:00Z,9999-01-01T00:00:00Z
+TRVZ,10,L,H,1,false,false,false,2009-04-14T22:00:00Z,2024-09-23T21:34:00Z
```

Comment:
Are you closing the LH* (1Hz) streams? I think we should keep them open no?

Reply:
@CBurton90 the LH is  not configured   on the datalogger (from the data point of view) 
no data since 2024.267 
FYI @Thomas-Benson